Amazon.QLDB.Driver/driver/QldbDriver.cs

```diff
@@ -211,12 +265,165 @@ public IEnumerable<string> ListTableNames()
         /// </summary>
         public void Dispose()
         {
-            this.sessionPool.Dispose();
+            if (!this.isClosed)
```

Comment:
If the pool is moved into the base, this can be too.

Reply:
Even if not, since the implementations are the same I think it makes sense to factor into the base and pass the collection?

Reply:
I'm not sure what you mean?

Dispose can't take any parameters.